src/assets/scripts/util/cookie-notification.js

```diff
@@ -0,0 +1,34 @@
+import $ from 'jquery';
+
+var cookieNotification = {
+    el: {},
+    init: () => {
+        cookieNotification.cacheSelectors();
+        cookieNotification.showCookieBanner();
+        cookieNotification.hideCookieBanner();
+    },
+    cacheSelectors: () => {
+        cookieNotification.el.$cookieWrapper = $('.cookie-wrapper');
+        cookieNotification.el.$cookieButton = $('.cookie-button');
+    },
+    showCookieBanner: () => {
+        cookieNotification.el.$cookieWrapper.show();
+          cookieNotification.el.$cookieWrapper.animate({
+            bottom: '0px',
+          }, 500, function() {
+        });
+    },
+    hideCookieBanner: () => {
```

Comment:
I like the method names but they are inconsistent in what they do. the show cookie banner is great but the hide cookie banner binds the event. put the binding in the init. 
something like 
init: () => {
....
cookieNotification.el.$cookieButton.on('click', hideCookieBanner);
...